pkg/cli/manifest/testdata/valid.yaml

```diff
@@ -0,0 +1,7 @@
+namespace: MyCompany.Resources
```

Comment:
Feel `namespace` is also a overloaded term. `name` might be better. I have updated the example in design spec. 

If this needs discussion, we can also make this change later on as we evolve the spec

Reply:
sure, will change to name.